### ISSUE TITLE
Completion screen

### ISSUE THREAD

**Issue by ScalaWilliam** [chuckha/ColorFlood#1]
Nice game. Would be good to see a completion screen to say "You've won!" and maybe even a local storage listing of previous scores.

**Comment by A1195** [comment]
Yes, Score with minimum possible color change to fill the screen

**Comment by chuckha** [comment]
Opening another ticket for the minimum possible color change to fill the screen. Just moved to an easier-to-work-with data structure to get this feature in place.
